### DISCUSSION TITLE
beta? unstable!

### DISCUSSION THREAD

**Discussion by I21b** [ArcaeaOffline/client-android#19] [General]
这个apk名unstable~~绷不住了~~，不应该是beta然后加上pre-release吗（）

**Comment by 283375** [comment]
典型历史问题，原来启动应用后会弹不稳定版提示屏幕，从 MIUI 那学的强制 10 秒确认，不同意就退出应用。本意是免得安装包传来传去，搞得有人以为是正式版，然后丢了数据过来骂我，那我不挺委屈的（，总不能认为所有人都懂 alpha/beta/rc 是什么意思嘛。

然后又想着稳定版不需要这个屏幕，所以弄了 `unstable` 这么个 build flavor，同时改图标和应用名（*简直是完美的免责声明，真佩服我自己*），编译时自行切换就行。

虽然现在这个屏幕从开屏糊脸挪到设置里去了，但原来留下的代码我懒得清了，设置页也是靠这个 flavor 来决定是否显示提示的，所以就这样吧（

**Reply by I21b** [reply]
> 然后又想着稳定版不需要这个屏幕，所以弄了 `unstable` 这么个 build flavor，同时改图标和应用名（*简直是完美的免责声明，真佩服我自己*），编译时自行切换就行。

这是真天才

`The app may 💥 at any time, and your data is under the risk of 💥 even irrecoverable.`
